Grand.Web/App_Data/Localization/Upgrade/EN_410_420.nopres.xml

```diff
@@ -449,6 +449,12 @@
   <LocaleResource Name="Admin.Catalog.Categories.Fields.HideOnCatalog.hint">
     <Value>Allow store owner to hide some categories on category page (subcategories before the list of products).</Value>
   </LocaleResource>
+  <LocaleResource Name="Admin.Catalog.Categories.Fields.Class">
```

Comment:
You've added this resources to the wrong file. It's file related for upgrade from 4.10 to 4.20 version of GrandNode. You should add this code to the upgrade file 4.50 to 4.60.

Reply:
And this one should be deleted...